documentation-site/routes.js

```diff
@@ -43,12 +43,16 @@ const routes = [
     text: 'Components',
     children: [
       {
-        text: 'Data Entry',
+        text: 'Basic Inputs',
```

Comment:
It says `Basic Inputs` but contains a button example below. We should split button example into a separate child of the 'Components' instead.

Reply:
the idea came for Fabric UI, and how they categorized components: https://developer.microsoft.com/en-us/fabric#/components/button

what do you think, would it make sense to follow that?

Reply:
Oh I see, `Inputs` might be confusing, or at least at first. Other suggestion here can be `Basic controls`. No objections to using the current name tho.